src/DrupalVersionChanger.php

```diff
@@ -453,25 +453,33 @@ private function updateComposerJsonWithWildcards(string $next_stable_version): v
         /** @var array<string, string> $composer_json */
         $composer_json = $this->composerJson->read();
         // Define core packages to update with specific version.
-        $core_packages = ['drupal/core-recommended', 'drupal/core-composer-scaffold', 'drupal/core-dev'];
+        $core_packages = [
+          'require' => [
+            'drupal/core-recommended',
+            'drupal/core-composer-scaffold',
+          ],
+          'require-dev' => [
+            'drupal/core-dev'
```

Comment:
This isn’t something that requires immediate action for this particular PR, but I was asking myself: Is `drupal/core-dev ` typically required in a Drupal project?

If the answer to that question is no, then we shouldn’t assume that the current site already has it installed. Upgrading a package not originally included in the project could lead to issues.

We could create a TODO task to discuss this further and determine the best course of action.

Thanks.

Reply:
Noted, and agreed. I'll add a follow-up issue to clean this up.